plugin/documents.py

```diff
@@ -378,10 +379,20 @@ def on_selection_modified_async(self) -> None:
     def on_post_save_async(self) -> None:
         # Re-determine the URI; this time it's guaranteed to be a file because ST can only save files to a real
         # filesystem.
-        self.set_uri(view_to_uri(self.view))
-        if self.view.is_primary():
-            for sv in self.session_views_async():
-                sv.on_post_save_async(self._uri)
+        uri = view_to_uri(self.view)
+        new_scheme = parse_uri(uri)[0]
+        old_scheme = parse_uri(self._uri)[0]
```

Comment:
Something like this would look nicer (to me at least):
```suggestion
        new_scheme, _ = parse_uri(uri)
        old_scheme, _ = parse_uri(self._uri)
```